master/reference/calicoctl/resources/policy.md

```diff
@@ -44,17 +44,17 @@ spec:
 | name | The name of the policy. |         | string |
 | annotations | Opaque key/value information to be used by clients. | | map |
 
-
 #### Spec
 
 | Field      | Description                                                                                                                                           | Accepted Values | Schema                | Default |
```

Comment:
The table columns are redundant (accepted values == schema) and the layout is compressed. At some point in the future, let's reduce the columns in this section -- similar to the fix for [798](https://github.com/projectcalico/calico/issues/798).